doc/releases/changelog-dev.md

```diff
@@ -200,6 +200,10 @@
 * Circuit cutting now performs expansion to search for wire cuts in contained operations or tapes.
   [(#2340)](https://github.com/PennyLaneAI/pennylane/pull/2340)
 
+* The `qml.draw` and `qml.draw_mpl` transforms are now located in the `drawer` module. They can still be
```

Comment:
Should this be considered a breaking change? It shouldn't have any user facing ramifications.